scripts/validate-deployment.sh

```diff
@@ -9,7 +9,7 @@ result=''
 
 until [[ "${retry}" -ge 10 ]]; do
   # Query Prometheus for daml_health_status >= 1
-  result=$(curl -sS --retry 10 --retry-delay 5 --retry-connrefused 'http://localhost:9090/api/v1/query?query=daml_health_status+%3E%3D+1')
+  result=$(curl -sS --retry 10 --retry-delay 10 --retry-connrefused 'http://localhost:9090/api/v1/query?query=daml_health_status+%3E%3D+1')
```

Comment:
Why not use the default exponential backoff?

Reply:
It's too slow somehow, we want to the test to pass as soon as possible?